Makefile

```diff
@@ -94,39 +90,24 @@ tag: ## Report the git tag that would be used for the images
 version: ## Report the version that would be put in the binary
 	@echo $(VERSION)
 
-
-fmt-check: ## Check all source code formatting
-	@if [ -n "$(shell $(BUILD_CMD) gofmt -l ${GO_FILES})" ]; then \
-	  $(BUILD_CMD) gofmt -s -e -d ${GO_FILES}; \
-	  exit 1; \
-	fi
-
-fmt:   ## Format all source code files
-	$(BUILD_CMD) gofmt -w -s ${GO_FILES}
+fmt: golangci-lint  ## Format all source code files
+	@$(BUILD_CMD) $(LINTER) run --fix ./
 
 golangci-lint: $(LINTER)
 $(LINTER):
-	mkdir -p hacks && cd hacks && (go mod init hacks || true) && go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.27.0
-
-golint:
-ifeq (, $(shell which golint))
-	go get -u golang.org/x/lint/golint
-endif
-
-lint: pkgs golangci-lint ## Lint the files
-	@$(BUILD_CMD) $(LINTER) run --disable-all --enable=golint ./ ./metal
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.44.2
```

Comment:
Update to use the latest version of golangci-lint, since 1.27.0 is quite a bit old.